src/file_info.rs

```diff
@@ -182,14 +182,10 @@ impl FileInfo<'_, '_> {
     }
 }
 
-fn _get_base_from_glob(idx: usize, glob: &'_ str) -> Result<PathBuf, ConfigError> {
-    let base = if let Some(b) = glob.split('*').next() {
-        b
-    } else {
-        return Err(ConfigError::AssetGlobInvalid(
-            idx,
-            "Glob may not start string",
-        ));
+fn _get_base_from_glob(_idx: usize, glob: &'_ str) -> Result<PathBuf, ConfigError> {
```

Comment:
Could you please change the function type to  `(&'_ str) -> PathBuf`? `Err` is no longer returned.

Reply:
Done in the latest push.